frontend/public/components/masthead.tsx

```diff
@@ -1,30 +1,155 @@
 import * as React from 'react';
+import { AboutModal } from 'patternfly-react';
+import * as classNames from 'classnames';
 
 import * as _ from 'lodash-es';
 import { Link } from 'react-router-dom';
 import * as okdLogoImg from '../imgs/okd-logo.svg';
 import * as ocpLogoImg from '../imgs/openshift-platform-logo.svg';
 import * as onlineLogoImg from '../imgs/openshift-online-logo.svg';
 import * as dedicatedLogoImg from '../imgs/openshift-dedicated-logo.svg';
+import * as rhLogoImg from '../imgs/redhat-logo-modal.svg';
+import * as okdModalImg from '../imgs/okd-logo-modal.svg';
 import { FLAGS, connectToFlags, flagPending } from '../features';
 import { authSvc } from '../module/auth';
 import { Dropdown, ActionsMenu } from './utils';
+import { openshiftHelpBase } from './utils/documentation';
+import { k8sVersion } from '../module/status';
 
 import { coFetchJSON } from '../co-fetch';
 import { SafetyFirst } from './safety-first';
 
 const developerConsoleURL = (window as any).SERVER_FLAGS.developerConsoleURL;
 
+class HelpMenu extends React.Component<HelpMenuProps, HelpMenuState> {
```

Comment:
Really should prefer using a functional component and connecting to Redux to fetch the Kubernetes version. It could be useful for other components. Or we could just add it server-side as a JS global to `index.html`.